jac-uk/admin
# #926 Barrister eligibility

## What's included?
* Add `Date called to the Bar` information under application qualifications section.
* Change `Has not completed pupillage` to `Has completed pupillage`.

Note: This PR is related to [apply #926 Barrister eligibility](https://github.com/jac-uk/apply/pull/927) and [jac-kit Add qualification constants and filters](https://github.com/jac-uk/jac-kit/pull/77). The version of `@jac-uk/jac-kit` will upgrade to `0.1.8`.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Go to Application Qualifications section.
2. Check if you can edit "Date called to the Bar", "Has completed pupillage", "Date completed pupillage", "Did not complete pupillage notes".

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/205704088-d3294f27-e05b-4d1f-89e4-1977eec1be96.mov

## Related permissions
- No permission changes required

---
PREVIEW:DEVELOP
